.scripts/chrome-unit.js

```diff
@@ -0,0 +1,21 @@
+const { spawn, exec } = require("child_process");
+const { join } = require("path");
+
+const webpackDevServer = spawn(join(__dirname, "../node_modules/.bin/ts-node"), [join(__dirname, "../testserver")], { shell: true })
+
+let mochaChromeRunning = false
+const webpackDevServerHandler = async (data) => {
+  if (!mochaChromeRunning) {
+    mochaChromeRunning = true
+    const mochaChrome = spawn(join(__dirname, "../node_modules/.bin/mocha-chrome"), ["http://localhost:3001"], { shell: true, stdio: "inherit" });
+    mochaChrome.on("exit", () => {
+      webpackDevServer.stderr.destroy();
+      webpackDevServer.stdout.destroy();
+      webpackDevServer.kill();
+    });
+  }
+}
+
+webpackDevServer.stderr.on('data', data => console.error(data.toString()));
```

Comment:
Could you put this into the webpackDevServerHandler function/lambda so that you only have a single 'data' handler?

Reply:
I think I do I want to log errors that come from the dev server, but not log most of the regular output (i.e. the webpack build report).